### DISCUSSION TITLE
Solution for cypress failed to make a connection to the Chrome DevTools Protocol

### DISCUSSION THREAD

**Discussion by SamikshaParkar** [cypress-io/cypress#16603] [Questions and Help]
Hello,
I need solution for cypress failed to make a connection to the Chrome DevTools Protocol in cypress
![connectionToChrome](https://user-images.githubusercontent.com/76738438/118945511-25b61680-b973-11eb-914a-f50fe2397604.png)
![image](https://user-images.githubusercontent.com/76738438/118945541-2c448e00-b973-11eb-9dd3-874e05767638.png)

**Comment by sandra-ouadghiri** [comment]
Hi,
Did you check the state of the port the error indicates? I would guess the port is used by another process.

**Reply by SamikshaParkar** [reply]
Okay I will check thanks for replying...

**Comment by sergavt** [comment]
It is not state of a port in my case at least.  Each time I run cypress script the new port number is being used for connection, besides these ports are not listening to any app either:
e.g.
>netstat -ano | find /i "54300"
  TCP    127.0.0.1:54305        127.0.0.1:54300        TIME_WAIT       0 

Here is the full trace:
=============================================================

Cypress failed to make a connection to the Chrome DevTools Protocol after retrying for 50 seconds.
This usually indicates there was a problem opening the Edge browser.
The CDP port requested was 56212.

Error: Could not find url target in browser about:blank. Targets were [{"targetId":"AF60BBD0B2DD967D1016994C8E0D656F","type":"service_worker","title":"Service Worker https://ntp.msn.com/edge/ntp/service-worker.js?bundles=latest&enableOfficeRoutes=true&riverAgeMinutes=2880&navAgeMinutes=2880&networkTimeoutSeconds=5&enableNetworkFirst=true&enableEmptySectionRoute=true&enableNavPreload=true&enableFallbackVerticalsFeed=true&noCacheLayoutTemplates=true","url":"https://ntp.msn.com/edge/ntp/service-worker.js?bundles=latest&enableOfficeRoutes=true&riverAgeMinutes=2880&navAgeMinutes=2880&networkTimeoutSeconds=5&enableNetworkFirst=true&enableEmptySectionRoute=true&enableNavPreload=true&enableFallbackVerticalsFeed=true&noCacheLayoutTemplates=true","attached":false,"canAccessOpener":false,"browserContextId":"74D9D0870712D3E61578C05BA272421D","pid":25592},{"targetId":"671D3C2526BB85DEC6E8CA5815E11F34","type":"service_worker","title":"Service Worker chrome-extension://bpgncafocbpieaeigfcookhgmeamglgo/background.js","url":"chrome-extension://bpgncafocbpieaeigfcookhgmeamglgo/background.js","attached":false,"canAccessOpener":false,"browserContextId":"74D9D0870712D3E61578C05BA272421D","pid":15892},{"targetId":"C704B6751D25B5D3D10E79D90C4989CD","type":"service_worker","title":"Service Worker https://res.cdn.office.net/midgard/versionless/sw.js","url":"https://res.cdn.office.net/midgard/versionless/sw.js","attached":false,"canAccessOpener":false,"browserContextId":"74D9D0870712D3E61578C05BA272421D","pid":23360},{"targetId":"81F1E6EB67907FCB25270FCF947E607A","type":"page","title":"New tab","url":"edge://newtab/","attached":false,"canAccessOpener":false,"browserContextId":"74D9D0870712D3E61578C05BA272421D","pid":0},{"targetId":"D601181B91DCD9D41605285F026448F5","type":"iframe","title":"https://business.bing.com/msb/r/msbdata_3a1e2491b096bc002a12.html","url":"https://business.bing.com/msb/r/msbdata_3a1e2491b096bc002a12.html","attached":false,"canAccessOpener":false,"browserContextId":"74D9D0870712D3E61578C05BA272421D","pid":0},{"targetId":"AAE0FA30C3BFE18F2049C1B47E991F33","type":"iframe","title":"https://res.cdn.office.net/midgard/versionless/feedhost_1690a693ae5298806b89.html?version=1.20231211.7.0&feedEdgeLoaderCatalogCacheVersion=2","url":"https://res.cdn.office.net/midgard/versionless/feedhost_1690a693ae5298806b89.html?version=1.20231211.7.0&feedEdgeLoaderCatalogCacheVersion=2","attached":false,"canAccessOpener":false,"browserContextId":"74D9D0870712D3E61578C05BA272421D","pid":0}]
    at <embedded>:4507:315879
    at async t (<embedded>:4507:315043)
    at async Object.open (<embedded>:4513:50294)
    at async Object.open (<embedded>:4513:25936)
    at async v.relaunchBrowser (<embedded>:4579:433407)
